subaru_global_2017.dbc

```diff
@@ -233,9 +235,9 @@ BO_ 802 ES_LKAS_State: 8 XXX
  SG_ FCW_Cont_Beep : 32|1@1+ (1,0) [0|1] "" XXX
  SG_ FCW_Repeated_Beep : 33|1@1+ (1,0) [0|1] "" XXX
  SG_ Throttle_Management_Activated : 34|1@1+ (1,0) [0|1] "" XXX
- SG_ Traffic_light_Ahead : 35|1@1+ (1,0) [0|1] "" XXX
  SG_ Right_Depart : 36|1@1+ (1,0) [0|3] "" XXX
  SG_ Signal5 : 37|27@1+ (1,0) [0|1] "" XXX
+ SG_ Traffic_light_Ahead : 35|1@1+ (1,0) [0|1] "" XXX
```

Comment:
better to keep the signal in increasing order of starting bit, especially because it's a little endian dbc.

Reply:
fixed the Traffic_Light_Ahead signal order in next commit